klippy/toolhead.py

```diff
@@ -543,6 +543,14 @@ def register_lookahead_callback(self, callback):
         last_move.timing_callbacks.append(callback)
     def note_kinematic_activity(self, kin_time):
         self.last_kin_move_time = max(self.last_kin_move_time, kin_time)
+    def note_move_queue_activity(self, mq_time):
+        if self.special_queuing_state == "Flushed":
```

Comment:
During homing and force move, the `special_queueing_state` is set to "Drip".
While it is not recommended to have some `pwm-tool` in its non-default state during homing, it might still happen (e.g. queued GCODE).

If it is not flushed in this mode, it will still have an active PWM-Tool during homing, but will fail either because of `maximum_mcu_duration` or, after homing, due to the delayed schedule (`timer_too_close`).

At least, it happened to me with only `update_move_time`. I am not sure whether updating the `last_kin_move_time` in "Drip"-mode will lead to updates, because `_update_move_time(kin_time)` did not.